docs/quarantine.md

```diff
@@ -0,0 +1,95 @@
+goal (0.95 ** 2 = 0.9025). And, for instance, 17 failing tests with 5% failure rate
+would lead to a terrible 41.81% passing rate of the whole suite (0.95 ** 17 = 0.4181).
+
+## Procedure
+
+In order to remove as much as possible the influence of changes in PRs to
+determine the stability of the suite, we will take into account only results
+from the periodics that run e2e tests from master, these jobs can be checked
+[on testgrid].
+
+In the following definitions we will consider test failures only when reported on
+testgrid in jobs where less than 5 tests failed, so that we don't take into
+account systemic failures caused for instance by an infrastructure problem.
+
+### Putting tests in quarantine
+
+In the following failure rates are calculating taking into account periodics and
+presubmits that are executed on merged code (on tide merge batches).
+
+A test must be put in quarantine when any of these conditions is met:
+* It has a failure rate higher than 5% in the last two weeks.
+* It has a failure rate higher than 20% in the last 3 days.
+
+A PR will be proposed on Mondays every two weeks with a batch of the tests that
+met the first condition. A PR can be proposed at any time for the tests that meet
+the second condition. In both cases the PR will add the text `[QUARANTINE]` to
+each test's description in the code. An email will be sent to the owners of the
+suspected tests.
+
+After the PR with the quarantine candidates is proposed there is a grace period
+of 2 days to prepare and land a fix for a test in the batch. If at least 5
+consecutive executions with the fix pass the test can be removed from the batch.
+
+Each quarantined test must have a team owner. The PR will add the text
+`[owner:@sig-{compute,network,storage}]` to each test's description.
+
+When a test marked with the [release-blocker] meets the conditions to be
+quarantined we will:
+* Create github issue with a comment `/release-blocker master` to ensure that
+the issue is addressed before a new release is cut.
+* Ensure that the github issue is assigned to an individual who will own bringing
+the blocker to completion within a quick time frame.
+
+### Getting tests out of quarantine
+
+The person assigned to each tracking ticket should propose a fix for the test or,
+after investigating the source of the errors, determine that the test itself
+doesn't need changes to be fixed (maybe the fix needs to be done on other parts
+of the code base or in a separate repo). In any case, the person assigned must
+communicate when the test is expected to be stable.
+
+A test must be put out of quarantine when:
+* It hasn't failed on any of the periodic lanes in the two weeks after the time
+indicated by the person assigned to bring the test back to the stable suite.
+
+The person assigned will propose a PR after the two weeks with successful executions
+has passed that will remove the text `[QUARANTINE]` from the test description in
+the code.
```

Comment:
A bit too long sentence, can you split it?

Reply:
Sure done.